app/views/partials/_edit_item.html.erb

```diff
@@ -48,6 +69,20 @@
     </div>
 
     <div class="form-group">
+      <%= f.label :available, "Item Not Currently Available?", class: "col-sm-2 control-label" %>
```

Comment:
Is "Item Not Currently Available?" the text for the checkbox? "Item Currently Available" might be easier to read at a glance, but you might need to change the default of the checkbox to fit the wording.